Add works and about section with button hover effects

MesakDuduCoder opened the pull request:
Added works section with projects displayed and button hover effects.
Added about section with button hover effects.

Felix45 commented on index.html:
```diff
@@ -132,5 +123,251 @@
+                            <div class="year"><p>2015</p></div>
+                        </div>
+                    </div>
+                    <h2 class="project-details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </h2>
```
- [ Required ] Please consider changing the project descriptions in your `works` section to a `p` tag, the best tag to use when marking up multi-line text is the `p` tag. The `hx` - `h1` - `h2`  tags are usually used to markup headings on a webpage and this is not a heading :smile:

Felix45 commented on index.html:
```diff
@@ -132,5 +123,251 @@
+                        <li class="tags-list"><a class="tags-text" href="">javaScript</a></li>
+                    </ul>
+                    <div class="actions">
+                        <button>
+                            <a href="">See&nbspProject</a>
+                        </button>
```
- [ Required ] The See project button can be either a `button` or `a` tag but using both is not good practice, I would like to suggest that you use the `button` tag. Please consider changing this in all the `See project` buttons in the works section.

Felix45 commented on index.html:
```diff
@@ -132,5 +123,251 @@
         </section>
     </header>
+
+    <main class="works">
```
- [ Required ] It would be better if you try as much as possible to stick with the colors provided on the Figma design, If you look at the background color around the `work-cards` you'll notice that the background color on your design doesn't match the background color on the Figma design. Please see the screenshot below for more details on this.

## Figma design
<img src="https://user-images.githubusercontent.com/16392046/221296415-a96f2ae0-e208-4419-ba80-ced9ac886e7d.png" width="375px" />

## Your design
<img src="https://user-images.githubusercontent.com/16392046/221296654-3942d035-4876-4346-83f5-8d273a340545.png" width="375px" />

Felix45 commented on index.html:
```diff
@@ -132,5 +123,251 @@
+
+    <section class="about-myself">
+        <div class="top">
+            <h1>
+                About <br>
+                Myself
+            </h1>
```
- [ Required ] A well-designed webpage usually has one `h1` tag which marks the most important title on the webpage, your webpage appears to have more than one which is not good practice. Please consider changing this `h1` tag to a `h2` tag to make your webpage semantically correct.

Felix45 commented on index.html:
```diff
@@ -132,5 +123,251 @@
+                About <br>
+                Myself
+            </h1>
+            <h2>
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </h2>
```
- [ Required ] Please consider changing the about myself description to a `p` tag, the best tag to use when marking up multi-line text is the `p` tag. The `hx` - `h1` - `h2`  tags are usually used to markup headings on a webpage and this is not a heading, please rectify this.

Felix45 commented on index.html:
```diff
@@ -132,5 +123,251 @@
+                    </ul>
+                </div>
+            </div>
+            <button class="normal-button" href=""><a>Get&nbspmy&nbspresume</a></button>
```
- [ Required ] The "Get my resume" button can be either a `button` or `a` tag but using both is not good practice, I would like to suggest that you use the `button` tag instead.

Felix45 commented on index.html:
```diff
@@ -132,5 +123,251 @@
+            <button class="normal-button" href=""><a>Get&nbspmy&nbspresume</a></button>
+        </div>
+
+        <ul class="skill-list">
```
- [ Required ] Good job designing this section you did well, however, I noticed some areas that if attended to will make your design even better. Please consider the following observations and adjust.
   
   - There should be a `border-bottom` below the `Frameworks` and `Skills` part 
   - There should be no `margin-left` on the `JavaScript` `CSS` and `HTML5` list
   - The clickable buttons on the list should not have a gray background color

## Figma design
<img src="https://user-images.githubusercontent.com/16392046/221300570-5713be97-d3a6-4599-907f-0d0563785e2e.png" width="375px" />

## Your design
<img src="https://user-images.githubusercontent.com/16392046/221300252-fff5bbf8-5bd2-4ee4-a8cb-a3ccfea6fe01.png" width="375px" />

Felix45 reviewed (changes requested):
Hi @MesakDuduCoder , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://camo.githubusercontent.com/56bcd754643682fcc078b66eaa7278dead6c9c5013595c875b4ddcbe0f0ef0aa/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d2e676966" />

## Highlights :heavy_check_mark: 
- [x] No linter errors :white_check_mark: 
- [x] Repository has a professional readme file :white_check_mark:  
- [x] GitHub flow was used correctly :white_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Kamba56 commented on index.html:
```diff
@@ -132,5 +123,251 @@
         </section>
     </header>
+
+    <section class="works">
```
- [ ] As requested by the previous reviewer, kindly change the background color of the work section to match the given template. You are expected to stick to design as much as possible. The background color on the figma template is `#e5e5e5` and your background is white.
### Figma
![image](https://user-images.githubusercontent.com/94551484/221348438-ef8bcbd8-d686-4efb-8658-93fa7b5d4d09.png)
### Your design
![image](https://user-images.githubusercontent.com/94551484/221348530-6c3d1d63-ddf3-47d5-bc64-01a1ec45d7b1.png)

Kamba56 reviewed (changes requested):
Hi @MesakDuduCoder, 

Great work on making the changes requested by a previous reviewer 👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed!👏👏👏
- [ ] Great UI design keep it up.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

MesakDuduCoder commented:
Fixed extra h1 tags to other tags and fixed border-bottom for Frameworks and Skills and also fixed background color of chevron icon buttons and background color of  works section.

divinecharlotte commented on index.html:
```diff
@@ -132,5 +123,251 @@
+                    <li>
+                        <img src="./icons/js.png" alt="">
+                        <h5>JavaScript</h5>                       
+                    </li>
+                    <li>
+                        <img src="./icons/html.png" alt="">
+                        <h5>HTML</h5>                       
+                    </li>
+                    <li>
+                        <img src="./icons/css.png" alt="">
+                        <h5>CSS</h5>                   
+                    </li>
+                </ul>
+            </li>
+            
+            <li class="skill-list-2">
+                <div class="skill-list-names">
+                    <h4>Frameworks</h4>
+                    <button>
+                        <div class="arrow-container">
+                            <svg width="8" height="15" viewBox="0 0 8 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <path fill-rule="evenodd" clip-rule="evenodd" d="M0.292893 0.439622C0.683417 0.0490974 1.31658 0.0490974 1.70711 0.439622L7.70711 6.43962C8.09763 6.83015 8.09763 7.46331 7.70711 7.85384L1.70711 13.8538C1.31658 14.2444 0.683417 14.2444 0.292893 13.8538C-0.0976311 13.4633 -0.0976311 12.8301 0.292893 12.4396L5.58579 7.14673L0.292893 1.85384C-0.0976311 1.46331 -0.09763 ...
+                            </svg>
+                        </div>
+                    </button>
+                </div>    
+            </li>
+            <li class="skill-list-2">
+                <div class="skill-list-names">
+                    <h4>Skills</h4>
+                    <button>
+                        <div class="arrow-container">
+                            <svg width="8" height="15" viewBox="0 0 8 15" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <path fill-rule="evenodd" clip-rule="evenodd" d="M0.292893 0.439622C0.683417 0.0490974 1.31658 0.0490974 1.70711 0.439622L7.70711 6.43962C8.09763 6.83015 8.09763 7.46331 7.70711 7.85384L1.70711 13.8538C1.31658 14.2444 0.683417 14.2444 0.292893 13.8538C-0.0976311 13.4633 -0.0976311 12.8301 0.292893 12.4396L5.58579 7.14673L0.292893 1.85384C-0.0976311 1.46331 -0.09763 ...
+                            </svg>
+                        </div>
+                    </button>
+                </div>    
+            </li>
+        </ul>
```
- [ ] as been requested by the previous reviewer consider removing the background color totally  you gave those arrows  so as to make your design look like the one provided in the Figma design
- your design 👇 
![new](https://user-images.githubusercontent.com/60146030/221367778-d2b71cff-1fa9-49c3-97d2-f79169eb8054.PNG)
- Figma design 👇 
![ss](https://user-images.githubusercontent.com/60146030/221367990-1cc169e5-cb29-4408-93f5-151b7cd77e0c.PNG)

divinecharlotte reviewed (changes requested):
# Hello @MesakDuduCoder   👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

![almost-there](https://media.giphy.com/media/ReyD4OdRiHeLDCRaEP/giphy.gif)

## Highlights 🍾  
- GOOD COMMIT MESSAGES 👍 
- PROFESSIONAL README 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

MesakDuduCoder commented:
Fixed button background color to match figma design.

huseyinbiyik reviewed (approved):
Hi @MesakDuduCoder,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Alt Text](https://media.giphy.com/media/wZjlCH43M3M0U/giphy.gif)



## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
